docs/kytConfig.md

```diff
@@ -37,6 +37,8 @@ modifyWebpackConfig: (baseConfig, options) => {
 }
 ```
 
+> **NOTE:** [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) is included in the Webpack configs' entry.main arrays by default. If you use `modifyWebpackConfig()` to override the entry properties and wish to keep using the polyfill, the new entries should go after the polyfill.
```

Comment:
Thanks, this is better, although I want to clarify that new entries can be added as needed but care only need to be taken when/if modifying `entry.main[]` in particular. 

What do you think of something like this?:

```
> **NOTE:** [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) is included in the Webpack configs' `entry.main[]` array by default. If you use `modifyWebpackConfig()` to modify `entry.main[]` and you wish to keep using the polyfill, be sure to add the new main entries _after_ the polyfill.
```